### DISCUSSION TITLE
Does flashlist support customized RefreshControl component?

### DISCUSSION THREAD

**Discussion by chj-damon** [Shopify/flash-list#598] [Q&A] [answered]
react-native `RefreshControl ` component is sometimes ugly from user's perspective, UI designers always want to replace it with some fancy customized style. I don't know if flashlist support this?

**Comment by r-hirsch** [comment, marked as answer]
Yes it does, see the below.

https://shopify.github.io/flash-list/docs/usage/#refreshcontrol

**Reply by chj-damon** [reply]
wow! this is amazing. I didn't notice that before.

**Reply by skizzo** [reply]
Are there any examples for using a custom RefreshControl? I can't get it to work, always receiving the error
```
Exception thrown while executing UI block: RCTScrollView may only contain a single subview, the already set subview looks like: <RCTView ....
```
